### DISCUSSION TITLE
Filter Release

### DISCUSSION THREAD

**Discussion by Achyuth072** [ImranR98/Obtainium#1869] [Q&A] [answered]
I have K-9 mail installed and it was fine before but recently they transitioned to thunderbird but it seems like they are still releasing K-9 versions alongside Thunderbird. But Obtanium keeps showing the thunderbird version which cant be installed as the packages different.
Repo: https://github.com/thunderbird/thunderbird-android

**Comment by DwainZwerg** [comment, marked as answer]
I had the same problem; a relatively simple solution: Enter “K” in the release title (in Obtainium)

**Reply by Achyuth072** [reply]
Thanks worked!

**Comment by ryosoftware** [comment]
When I try the propossed workaround (add "K" at the release title) Obtainium says "no valid version found".

**Reply by DwainZwerg** [reply]
Strange, it works for me. Can you please try the config from the website to see if you have entered it in the wrong position? If that doesn't work, can you please remove the app from Obtainium and add it again from the website?

**Reply by DarkCrypt** [reply]
Very strange, I've been trying to reproduce this issue and fail. I can't even reproduce the same error message.

There's still something wrong with your config. DwainZwerg's fix is 100% correct and the config from Obtainium's Config website is also correct.

**Comment by ryosoftware** [comment]
Hello,
I've removed then added again, and now Obtainium shows "not installed". I think it's due to thunderbird and k9 shares the same repo...

![image](https://github.com/user-attachments/assets/01a2d39b-646b-4e5b-994f-e30ffc0aeb33)
![image](https://github.com/user-attachments/assets/8c363b75-402a-4abb-81af-494278773b00)
![image](https://github.com/user-attachments/assets/45e16cf8-811b-446f-82bb-ae5bfdbb2aa2)
![image](https://github.com/user-attachments/assets/2070c99a-8c90-4121-8a81-9548b605b390)

and this occurs when I include the "K" at release title:

![image](https://github.com/user-attachments/assets/2982c3e9-2364-4b54-92cd-3cb860028ad1)
![image](https://github.com/user-attachments/assets/84c731d9-ebe8-43fd-883b-207e2661f44f)

Thanks

**Reply by DarkCrypt** [reply]
> I think it's due to thunderbird and k9 shares the same repo...

More than 1 different app within a repo is not the issue, other repos do this also and Obtainium can filter perfectly. The issue is your config is still wrong.

Try the config taken from Obtainium's Config website: Import App -> [K-9 Mail](https://apps.obtainium.imranr.dev/redirect?r=obtainium://app/%7B%22id%22%3A%22com.fsck.k9%22%2C%22url%22%3A%22https%3A%2F%2Fgithub.com%2Fthunderbird%2Fthunderbird-android%22%2C%22author%22%3A%22thunderbird%22%2C%22name%22%3A%22K-9%20Mail%22%2C%22preferredApkIndex%22%3A0%2C%22additionalSettings%22%3A%22%7B%5C%22includePrereleases%5C%22%3Afalse%2C%5C%22fallbackToOlderReleases%5C%22%3Atrue%2C%5C%22filterReleaseTitlesByRegEx%5C%22%3A%5C%22K-9%5C%22%2C%5C%22filterReleaseNotesByRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22verifyLatestTag%5C%22%3Afalse%2C%5C%22dontSortReleasesList%5C%22%3Afalse%2C%5C%22useLatestAssetDateAsReleaseDate%5C%22%3Afalse%2C%5C%22releaseTitleAsVersion%5C%22%3Afalse%2C%5C%22trackOnly%5C%22%3Afalse%2C%5C%22versionExtractionRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22matchGroupToUse%5C%22%3A%5C%22%5C%22%2C%5C%22versionDetection%5C%22%3Atrue%2C%5C%22releaseDateAsVersion%5C%22%3Afalse%2C%5C%22useVersionCodeAsOSVersion%5C%22%3Afalse%2C%5C%22apkFilterRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22invertAPKFilter%5C%22%3Afalse%2C%5C%22autoApkFilterByArch%5C%22%3Atrue%2C%5C%22appName%5C%22%3A%5C%22%5C%22%2C%5C%22shizukuPretendToBeGooglePlay%5C%22%3Afalse%2C%5C%22allowInsecure%5C%22%3Afalse%2C%5C%22exemptFromBackgroundUpdates%5C%22%3Afalse%2C%5C%22skipUpdateNotifications%5C%22%3Afalse%2C%5C%22about%5C%22%3A%5C%22K-9%20Mail%20is%20an%20open-source%20email%20client%20for%20Android%20made%20by%20Mozilla.%5C%22%7D%22%2C%22overrideSource%22%3Anull%7D)

**Reply by ryosoftware** [reply]
> > I think it's due to thunderbird and k9 shares the same repo...
> 
> More than 1 different app within a repo is not the issue, other repos do this also and Obtainium can filter perfectly. The issue is your config is still wrong.
> 
> Try the config taken from Obtainium's Config website: Import App -> [K-9 Mail](https://apps.obtainium.imranr.dev/redirect?r=obtainium://app/%7B%22id%22%3A%22com.fsck.k9%22%2C%22url%22%3A%22https%3A%2F%2Fgithub.com%2Fthunderbird%2Fthunderbird-android%22%2C%22author%22%3A%22thunderbird%22%2C%22name%22%3A%22K-9%20Mail%22%2C%22preferredApkIndex%22%3A0%2C%22additionalSettings%22%3A%22%7B%5C%22includePrereleases%5C%22%3Afalse%2C%5C%22fallbackToOlderReleases%5C%22%3Atrue%2C%5C%22filterReleaseTitlesByRegEx%5C%22%3A%5C%22K-9%5C%22%2C%5C%22filterReleaseNotesByRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22verifyLatestTag%5C%22%3Afalse%2C%5C%22dontSortReleasesList%5C%22%3Afalse%2C%5C%22useLatestAssetDateAsReleaseDate%5C%22%3Afalse%2C%5C%22releaseTitleAsVersion%5C%22%3Afalse%2C%5C%22trackOnly%5C%22%3Afalse%2C%5C%22versionExtractionRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22matchGroupToUse%5C%22%3A%5C%22%5C%22%2C%5C%22versionDetection%5C%22%3Atrue%2C%5C%22releaseDateAsVersion%5C%22%3Afalse%2C%5C%22useVersionCodeAsOSVersion%5C%22%3Afalse%2C%5C%22apkFilterRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22invertAPKFilter%5C%22%3Afalse%2C%5C%22autoApkFilterByArch%5C%22%3Atrue%2C%5C%22appName%5C%22%3A%5C%22%5C%22%2C%5C%22shizukuPretendToBeGooglePlay%5C%22%3Afalse%2C%5C%22allowInsecure%5C%22%3Afalse%2C%5C%22exemptFromBackgroundUpdates%5C%22%3Afalse%2C%5C%22skipUpdateNotifications%5C%22%3Afalse%2C%5C%22about%5C%22%3A%5C%22K-9%20Mail%20is%20an%20open-source%20email%20client%20for%20Android%20made%20by%20Mozilla.%5C%22%7D%22%2C%22overrideSource%22%3Anull%7D)

Thanks, is working now!
